tests/tabular/checks/integrity/string_mismatch_test.py

```diff
@@ -154,6 +159,7 @@ def test_condition_percent_variants_no_more_than_pass():
     # Assert
     assert_that(result, has_items(
         equal_condition_result(is_pass=True,
+                               details='Passed for 1 string columns',
```

Comment:
Here and in other places, we say "columns" even when the number is one.
Maybe add some sort of if so it won't be plural? I know it's annoying, and users will get why it's like that, but it's more professional like this.
Can also create a common method for that  - 

Suggestion:
def add_plural(num: int, postfix: str='s'):
    return postfix if num == 1 else ''

and then use as:
f'Found {num_columns} column{add_plural(num_columns)} ... '
and you can change the postfix into "es" where relevant.
Not great, but can change the experience a bit